tests/integration/core/remote_operations.py

```diff
@@ -928,7 +928,7 @@ def clear_ha(self, server_list):
 
             with open(clear_ha_script_file, "r") as clear_ha_script:
                 result = self._ssh_address(
-                    address, "ring1_iface=%s\n%s" % (server["corosync_config"]["ring1_iface"], clear_ha_script.read())
+                    address, "ring0_iface=%s\n%s" % (server["corosync_config"]["ring1_iface"], clear_ha_script.read())
```

Comment:
Why it it using ring1_iface instead of ring0_iface? Would it not be like this:
```
address, "ring0_iface=%s\n%s" % (server["corosync_config"]["ring0_iface"], clear_ha_script.read())
```